torch/distributed/_tensor/dispatch.py

```diff
@@ -16,17 +17,22 @@
     OutputSpecType,
 )
 from torch.distributed._tensor.placement_types import DTensorSpec
-from torch.distributed._tensor.random import (
-    _get_rng_offset,
-    is_rng_supported_mesh,
-    set_post_op_offset,
-    set_pre_op_offset,
-)
+from torch.distributed._tensor.random import is_rng_supported_mesh
 from torch.distributed._tensor.redistribute import redistribute_dtensor
 from torch.distributed._tensor.sharding_prop import ShardingPropagator
 from torch.utils._pytree import tree_flatten, tree_unflatten
 
 
+def _is_random_op(op):
+    aten = torch.ops.aten
+    random_ops = [
+        aten.native_dropout.default,
+        aten.normal_.default,
+        aten.uniform_.default,
+    ]
```

Comment:
why do we create this list within the function? Any particular reason?

Reply:
it's simply to help check if an operator is a random op. we can extend this list when we are going to support more random operators.

Reply:
No, my question is why do you make this list a local variable inside a function?

Reply:
n00b question on python: if I make it a global variable, will there be a performance gain? Is this the alternative approach you propose?
```
random_ops = [...]

def _is_random_op(op):
    return op in random_ops
```

Reply:
Well, at least:
```
return op in [
        aten.native_dropout.default,
        aten.normal_.default,
        aten.uniform_.default,
]
```
?

And make aten = torch.ops.aten a const?